targets/TARGET_TT/TARGET_TT_M4G9/device/TOOLCHAIN_GCC_ARM/tmpm4g9f15fg.ld

```diff
@@ -10,6 +10,12 @@
   #define MBED_APP_SIZE 0x180000
 #endif
 
+#if !defined(MBED_BOOT_STACK_SIZE)
```

Comment:
Can you please split this to own commit ? It's not related to NC gpio configuration